addon/components/uk-modal.js

```diff
@@ -60,13 +60,19 @@ export default Component.extend({
   },
 
   didReceiveAttrs() {
-    scheduleOnce("afterRender", () => {
-      this.visible ? this.modal.show() : this.modal.hide();
-    });
+    scheduleOnce("afterRender", this.initVisible);
   },
 
   willDestroyElement() {
     this.modal.$el.parentNode.removeChild(this.modal.$el);
     this.set("modal", null);
+  },
+
+  initVisible() {
```

Comment:
Not that I'm against this change. But why did you do this? Does this fix anything technical?

Reply:
https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-incorrect-calls-with-inline-anonymous-functions.md This rule is now enabled per default

Reply:
Thank you!